frontend/.prettierignore

```diff
@@ -0,0 +1,2 @@
+graphql
+API.ts
```

Comment:
if you add it here. you need to disable your prettier on save. Otherwise, you will still override it. like you have done in the files below